ONSdigital/eq-survey-runner
# EQ 1 JSON format

**What**
Implementing the integration with RRM, so extracting the required information from the JWT. Also converting the users responses to the required JSON format for the data integration team.

**How to test**
1. Run the application
2. Complete a survey
3. In the logs locate the JSON response being sent downstream and check the format

**Who can review**
Anyone apart from @warrenbailey
